modules/core/include/opencv2/core/cuda.hpp

```diff
@@ -808,23 +808,23 @@ class CV_EXPORTS TargetArchs
     @param major Major compute capability version.
     @param minor Minor compute capability version.
      */
-    static bool has(int major, int minor);
-    static bool hasPtx(int major, int minor);
-    static bool hasBin(int major, int minor);
-
-    static bool hasEqualOrLessPtx(int major, int minor);
-    static bool hasEqualOrGreater(int major, int minor);
-    static bool hasEqualOrGreaterPtx(int major, int minor);
-    static bool hasEqualOrGreaterBin(int major, int minor);
+    CV_WRAP static bool has(int major, int minor);
```

Comment:
"static" is not properly handled in Python bindings generator: `METH_STATIC` should be used.

Reply:
`METH_STATIC` is now utilized